plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergV2.java

```diff
@@ -210,6 +226,42 @@ public void testV2TableWithEqualityDelete()
         assertQuery("SELECT nationkey, comment FROM " + tableName, "SELECT nationkey, comment FROM nation WHERE regionkey != 1");
     }
 
+    @Test
+    public void testV2TableWithEqualityDeleteVarbinary()
```

Comment:
do we need 5 more tests in this class?
or would having the one in BaseIcebergConnectorTest.java  be enough?

Reply:
There is a path: `io.trino.plugin.iceberg.delete.EqualityDeleteFilter#readEqualityDeletes` &#8594; `io.trino.plugin.iceberg.delete.TrinoRow` &#8594; `io.trino.plugin.iceberg.IcebergPageSink#getIcebergValue`

These tests check this path

Reply:
understood, but `IcebergPageSink#getIcebergValue` could actually be unit tested.
varbinary transform should be also tested (good)

i proposed not to add these other tests

Reply:
Ok, I removed them